web-components/package.json

```diff
@@ -90,7 +90,7 @@
     "@types/sortablejs": "^1.10.6",
     "@types/webpack": "^4.41.7",
     "@types/webpack-dev-server": "^3.10.0",
-    "@types/webpack-merge": "^4.1.5",
+    "@types/webpack-merge": "^5.0.0",
```

Comment:
@Budhia What's up with the upgrade? Were you hitting issues with your build that should go library wide?

Reply:
Yes. While I was trying to setup the repo the dist:dev command was giving issues related to the webpack version. So upgrading it solved the scenario.